### ISSUE TITLE
polyfill issues in IE 11 on Win 10

### ISSUE THREAD

**Issue by prasadmamidi7** [angular/angular-cli#8834]
Even after enabling required poly fills for IE  application is not working in IE 11 on Win 10. After some investigation it was found that the issues is caused due to zone.js. If I comment the import statement for zone.js in polyfills.ts and include the zone.js in angular-cli.json under scripts sections it works as expected.

- I would like to know is this a know bug or issue 
- If I include zone.js in angular-cli.jsos under scripts sections, On production build where this script will be bundled i.e. vendor.bundle.js or some thing else.

### Versions
```
Angular CLI: 1.6.0
Node: 8.9.2
OS: win32 x64
Angular: 5.1.0
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

@angular/cli: 1.6.0
@angular-devkit/build-optimizer: 0.0.35
@angular-devkit/core: 0.0.22
@angular-devkit/schematics: 0.0.41
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.9.0
@schematics/angular: 0.1.10
@schematics/schematics: 0.0.10
typescript: 2.6.2
webpack: 3.10.0


```

### Repro steps
*  Create a sample application with angular cli 1.6.0
*  Make sure poly filles for IE is uncommented and build with --prod switch
*   deploy on win 10 and launch the application in IE 11 on win 10

### Observed behavior
```
If I include the zone.js in angular-cli.json under scripts sections it works as expected.
```

**Comment by pieterlouw1974** [comment]
I get Syntax error

eval("Object.defineProperty(__webpack_exports__, \"__esModule\", { value: true });\n/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__angular_core__ = __webpack_require__(\"../../../core/esm2015/core.js\");\n/* harmony import */ var __WEBPACK_IMPORTED_MODULE_1__angular_platform_browser_dynamic__ = __webpack_require__(\"../../../platform-browser-dynamic/esm2015/platform-browser-dynamic.js\");\n/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2__app_app_module__ = __webpack_require__(\"../../../../../src/app/app.module.ts\");\n/* harmony import */ var __WEBPACK_IMPORTED_MODULE_3__environments_environment__ = __webpack_require__(\"../../../../../src/environments/environment.ts\");\n\r\n\r\n\r\n\r\nif (__WEBPACK_IMPORTED_MODULE_3__environments_environment__[\"a\" /* environment */].production) {\r\n    Object(__WEBPACK_IMPORTED_MODULE_0__angular_core__[\"enableProdMode\"])();\r\n}\r\nObject(__WEBPACK_IMPORTED_MODULE_1__angular_platform_browser_dynamic__[\"a\" /* platformBrowserDynamic */])().bootstrapModule(__WEBPACK_IMPORTED_MODULE_2__app_app_module__[\"a\" /* AppModule */])\r\n    .catch(err => console.log(err));\r\n//# sourceURL=[module]\n//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJzb3VyY2VzIjpbImM6L0Rldl9Tb3VyY2UvWFRpbWVWMTEvc3JjL21haW4udHMiXSwibmFtZXMiOltdLCJtYXBwaW5ncyI6Ijs7Ozs7QUFBK0M7QUFDNEI7QUFFOUI7QUFDWTtBQUV6RCxFQUFFLENBQUMsQ0FBQyw4RUFBVyxDQUFDLFVBQVUsQ0FBQyxDQUFDLENBQUM7SUFDM0IscUVBQWMsRUFBRSxDQUFDO0FBQ25CLENBQUM7QUFFRCx5R0FBc0IsRUFBRSxDQUFDLGVBQWUsQ0FBQyxrRUFBUyxDQUFDO0tBQ2hELEtBQUssQ0FBQyxHQUFHLElBQUksT0FBTyxDQUFDLEdBQUcsQ0FBQyxHQUFHLENBQUMsQ0FBQyxDQUFDIiwiZmlsZSI6Ii4uLy4uLy4uLy4uLy4uL3NyYy9tYWluLnRzLmpzIiwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IHsgZW5hYmxlUHJvZE1vZGUgfSBmcm9tICdAYW5ndWxhci9jb3JlJztcbmltcG9ydCB7IHBsYXRmb3JtQnJvd3NlckR5bmFtaWMgfSBmcm9tICdAYW5ndWxhci9wbGF0Zm9ybS1icm93c2VyLWR5bmFtaWMnO1xuXG5pbXBvcnQgeyBBcHBNb2R1bGUgfSBmcm9tICcuL2FwcC9hcHAubW9kdWxlJztcbmltcG9ydCB7IGVudmlyb25tZW50IH0gZnJvbSAnLi9lbnZpcm9ubWVudHMvZW52aXJvbm1lbnQnO1xuXG5pZiAoZW52aXJvbm1lbnQucHJvZHVjdGlvbikge1xuICBlbmFibGVQcm9kTW9kZSgpO1xufVxuXG5wbGF0Zm9ybUJyb3dzZXJEeW5hbWljKCkuYm9vdHN0cmFwTW9kdWxlKEFwcE1vZHVsZSlcbiAgLmNhdGNoKGVyciA9PiBjb25zb2xlLmxvZyhlcnIpKTtcblxuXG5cbi8vIFdFQlBBQ0sgRk9PVEVSIC8vXG4vLyBjOi9EZXZfU291cmNlL1hUaW1lVjExL3NyYy9tYWluLnRzIl0sInNvdXJjZVJvb3QiOiJ3ZWJwYWNrOi8vLyJ9\n//# sourceURL=webpack-internal:///../../../../../src/main.ts\n")

**Comment by prasadmamidi7** [comment]
Hi

After further investigation I found that the problem is not caused due to zone.js whether it is included in the polyfills.ts or angular-cli.json file. But this was due to the below registry entry "HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Internet Explorer\Restrictions\NoNavButtons". which was set to 1.

Same issue referred in this link: https://github.com/angular/zone.js/issues/502

Now this has become a major blocker for our application as we do not have control on this registry entry. I would to like to know 
- why this error is thrown. ( I am assuming the library is trying to set the navigation url location information on to the nav buttons of the browser.)
- Is there any switch which will check this setting before pushing any state to these buttons or any other solution which may enable my application to work even when this registry entry is set to 1. 



Regards
Prasad Mamidi
